live-examples/css-examples/basic-box-model/contain.html

```diff
@@ -16,11 +37,11 @@
 
 <div id="output" class="output large hidden">
     <section id="default-example" class="default-example">
-        <div class="example-container">
-            <div id="example-element" class="transition-all">
-                <div class="floating">Floating Element</div>
-                The Goldfish is a species of domestic fish best known for its bright colors and patterns.
-            </div>
+        <div id="example-element" class="card">
+            <h2>Card Title</h2>
+            <p>The element will not grow or shrink to fit its content when size containment is applied.</p>
+            <p>This paragraph overflows the card in such cases.</p>
```

Comment:
Can we make this text more generic to cover any 'contain' value? Currently, it is specific to "contain:style" and that might be confusing when other 'contain' options are selected. Maybe use the original 'Goldfish' text? 🙂 

```suggestion
            <p>The element will not grow or shrink to fit its content when size containment is applied.</p>
            <p>This paragraph overflows the card in such cases.</p>
```

Also, do we want to demo "contain: content;"? The very original example had that option in the interactive example.

Reply:
> Maybe use the original 'Goldfish' text?

Good, I will replace it with this to make it more generic.

>do we want to demo "contain: content;"? 

Some background:

* `content` is shorthand for a combination of `layout paint style`
* `strict` is shorthand for `size layout paint style`

We may as well add `content` as we show off `strict`, too.

Reply:
I don't think it's a good idea to add 1 more value in there, because it will make scrollbar be shown:
![image](https://user-images.githubusercontent.com/100634371/204011059-b1f27ac9-3d72-4a60-b14e-19b160111d98.png)
Also value `content` has exactly the same visual appearance as value `paint` in this example.

Reply:
Actually, I think it makes sense to just demo `strict` instead of including `content`, there's no `style` (counters / quotes) in there so it's not really showing what the benefit is.

Reply:
Good, thanks.

>content has exactly the same visual appearance as value paint in this example.

Yeah `content` is `paint` and `layout` so I think we can drop it in favor of `strict`.